addons/website_slides/views/website_courses_templates.xml

```diff
@@ -0,0 +1,623 @@
+                        <div>
+                            <h1 class="font-weight-normal" t-field="channel.name"/>
+                            <div style="width:50%;">
+                                <p  class="d-block" t-field="channel.description"/>
+                            </div>
+                            <div>
+                                <a t-if="user.partner_id not in channel.partner_ids" t-attf-href="/slides/channel/enroll?channel_id=#{channel.id}"><button  style="width:25%;" class="btn btn-primary font-weight-bold">ENROLL</button></a>
+                                <a t-if="user.partner_id in channel.partner_ids and len(channel.slide_ids) > 0" t-attf-href="/slides/slide/#{slug(channel.slide_ids[0])}?fullscreen=1"><button  style="width:25%;" class="btn btn-primary font-weight-bold">Start Course</button></a>
+                            </div>
+                            <t t-if="user == channel.user_id" t-call="website_rating.rating_widget_stars_static">
+                                    <t t-set="rating_avg" t-value="channel.rating_avg"/>
+                                    <t t-set="rating_count" t-value="channel.rating_count"/>
+                                </t>
+                            <p t-else="" class="row mt8">
+                                <t t-call="website_rating.rating_stars_static_popup_composer">
+                                    <t t-set="rating_avg" t-value="rating_avg"/>
+                                    <t t-set="rating_total" t-value="rating_count"/>
+                                    <t t-set="object" t-value="channel"/>
+                                    <t t-set="hash" t-value="message_post_hash"/>
+                                    <t t-set="pid" t-value="message_post_pid"/>
+                                    <t t-set="default_message_id" t-value="last_message_id"/>
+                                    <t t-set="default_message" t-value="last_message"/>
+                                    <t t-set="default_rating_value" t-value="last_rating_value"/>
+                                    <t t-set="force_submit_url" t-value="'/slides/mail/update_comment' if last_message_id else False"/>
+                                </t>
+                            </p>
+                        </div>
+                        <div class="d-flex align-items-end">
+                            <div class="course-progress-bar">
+                                <div class="progress-gauge m-0 oe_slide_js_progress_bar" t-attf-completion="#{channel.completion}"/>
+                            </div>
+                            <i class="fa fa-trophy ml-2 mb-0 p-0" style="font-size:1.8rem; color: #cdcdcd;"></i>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div>
+            <div class="container mt-4">
+                <ul class="nav nav-tabs container" style="border:none !important " id="myTab" role="tablist">
```

Comment:
Weird ids "myTab" and "myTabContent".

Reply:
Probably old copypasta from bootstrap. Will rename that